### ISSUE TITLE
not training my subject? 

### ISSUE THREAD

**Issue by Maki9009** [d8ahazard/sd_dreambooth_extension#580]
im very confused, am I supposed to caption my images, cuz they're just random numbers? or something? 

because I'm training a person and non of the training samples or models seem to give me my subject at all.... not sure if its a conversation script error. like I've managed to train this person on shivams google colab. But I can't with this extension.. it doesn't seem to be training my subject or something. also I'm confused it trains 0 and says completed.. then trains 5 and says completed.. and then continues training to the steps I selected. but I'm not sure if its training my subject or not

I'm running the Auto1111 on PaperSpaces

**Comment by Alice2O3** [comment]
Same here, I'm training style based on novelai-leak and I can't get decent results. I'm trying to train an artist's artstyle (without class images) and I got this result:

![Webui_fail](https://user-images.githubusercontent.com/100117897/208357139-60ef1ff1-0b31-4f0d-9c12-e1e68de7afda.png)

I have tried many approaches:
1. Reload vae.pt inside leaked anime-full-latest-pruned.ckpt model
2. Tweaking Instance prompt and Instance token

However, when I used lora models, the result is significantly better when using approach 1. However, no matter how I use 1 and 2, the results are always like the picture above, way worse than the original diffusers script or the older version (before [filewords] feature added).

The problem may come from many possibilities:
1. Problems parsing [filewords], instance prompt and instance token
2. Problems loading vae.pt (it resolves problem with lora models)
3. Problems with the webui itself

**Comment by Maki9009** [comment]
![Capture](https://user-images.githubusercontent.com/62298209/208406025-258282fb-1e77-4dd0-b30e-5555b4679956.PNG)
is this normal when training? to do 0 then 5 and then start training? 

for me it doesn't seem like it even attempts to understand who my token is, like completely ignores them

**Comment by nanafy** [comment]
https://github.com/d8ahazard/sd_dreambooth_extension/discussions/577

not the only one, multiple people have already flagged this in discussion. Its simply not working

**Comment by Maki9009** [comment]
@nanafy i managed to get it to train my subject... you have to write a prompt like photo of Token Class

**Comment by nanafy** [comment]
Sick! I am trying again as well. I probably have been doing something wrong. There's just so many settings :D

**Comment by Maki9009** [comment]
@nanafy keep in mind though it might overfit... it seems to be really good at recreating images similar to the dataset.. you might need to prompt quiet a bit for it to mix with styles.. but it works well when it does. 

I'm testing now if it merges with models which seems like it does well kinda, seems better than other training methods also but idk still testing...

**Comment by nanafy** [comment]
What settings did you use?

**Comment by Maki9009** [comment]
Clicked on the performance wizard, 13 images, 150 for training samples based on training images. I put the token I wanted and the class word I wanted. Then prompt : Photo of token class / Class prompt : Photo of class 

Didn't change anything else, learning rate was default. 

300 class images 

Now keep in mind idk why but this makes the most god damn robust overfit model of a the person you trained lol... the model will work but give you very similar images to ur training images. To counter that you need long Strong good prompts to make it mix well with styles... 

But I also found out idk why but these models are good for merging with style models. 

Like iv merged style models with these and I gave my A model literally 30% and it was still good compared to when I merged with joepenna or shivams models. 

For merging. 

A : Dreambooth model from Auto111
B : Style model... better with fine tuned models but works well with some dreambooth models. 
C : 1.5 model 

Split set it to what works best... between 0.45 - 0.6 seem to give best of both worlds. 

Add difference. 

And yeah that's it still you'll need good prompt.. maybe not as long as unmerged. But idk why but these models merge pretty well compared to other dreambooth methods I tried. 

The only model that didn't merge well for me was Analog but I didn't test it much so idk

**Comment by nanafy** [comment]
That's so crazy I have the exact opposite problem.
I literally cannot get anything meaningful, check out my post
https://github.com/d8ahazard/sd_dreambooth_extension/discussions/595

I am going to try exactly what you just said. I feel like there is something odd happening but no one else seems to be really saying anything. I am starting to think I am crazy

**Comment by Maki9009** [comment]
@nanafy did it work for u?

**Comment by nanafy** [comment]
No, I trained for 10000 steps and got pure garbage. I re-installed Auto and the extension from scratch. Literally from scratch. All the site packages, everything deleted.  I never tried the new repo after that, I moved to a slightly older version of the newest one that still has LORA. I was able to train on that one but i noticed my prompting was not adhering well at all. So then I moved back to an even older version with 0.7.2 diffusers that still supports filewords for fine-tuning and its working like a charm. Maybe will try the new repo but I've wasted literally so much time training with it, that I need a break from trying it again lol. I took down my post 595 because no one commented. So I just assumed no one else has this issue.

**Comment by nanafy** [comment]
From my experience, the prompting is dependent on the diffusers. These 0.10 diffusers might just suck. I've noticed the biggest changer in image quality is the diffuser version. 0.7.2 seems to be pretty good.

**Comment by Maki9009** [comment]
Hmm not really sure tbh what version I am using.. cuz I'm running all this on paperspace, so it's usually never kinda updated. But these models merge amazingly... still didn't test training a style on it but I shall try and let you know... if you want I can try the style ur trying to get... if u can provide me ur dataset, also try to use shivams repo maybe that will work also

**Comment by bananasss00** [comment]
Try setting learning rate = 0.0009, trains fast on last commit

Settings from this video:
https://youtu.be/gw2XQ8HKTAI

**Comment by d8ahazard** [comment]
New version has been pushed, so please update and try again.

Closing to keep things tidy.
